src/rpc/blockchain.cpp

```diff
@@ -62,53 +62,70 @@ static Mutex cs_blockchange;
 static std::condition_variable cond_blockchange;
 static CUpdatedBlock latestblock GUARDED_BY(cs_blockchange);
 
-extern void TxToJSON(const CTransaction& tx, const uint256 hashBlock, CTxMemPool& mempool, llmq::CChainLocksHandler& clhandler, llmq::CInstantSendManager& isman, UniValue& entry);
+extern void TxToJSON(const CTransaction& tx, const uint256 hashBlock, CTxMemPool& mempool, CChainState& active_chainstate, llmq::CChainLocksHandler& clhandler, llmq::CInstantSendManager& isman, UniValue& entry);
 
-NodeContext& EnsureNodeContext(const CoreContext& context)
+NodeContext& EnsureAnyNodeContext(const CoreContext& ctx)
```

Comment:
I renamed it to `ctx` to match with [`context.h`](https://github.com/dashpay/dash/blob/92e5f9f35d8a099ecf14053ade68f13e4cb69175/src/context.h#L27) but realizing that this would cause more trouble than it's worth with cherry-pick conflicts down the line, it'll be reverted.